lib/read-wasm.js

```diff
@@ -0,0 +1,40 @@
+if (typeof fetch === "function") {
+  // Web version of reading a wasm file into an array buffer.
+
+  let mappingsWasmUrl = null;
+
+  module.exports = function readWasm() {
+    if (typeof mappingsWasmUrl !== "string") {
+      throw new Error("You must provide the URL of lib/mappings.wasm by calling " +
+                      "SourceMapConsumer.initialize({ 'lib/mappings.wasm': ... }) " +
```

Comment:
`initialize()` accepts just string though?

Reply:
The public `SourceMapConsumer.initialize` function takes an object, the interal `initialize` function in `lib/read-wasm.js` just takes a string.